wiki/Katu/id.md

```diff
@@ -0,0 +1,66 @@
+---
+tags:
+  - "100"
+  - katsu
+---
+
+# Katu
+
+*Lihat juga: [Geki](/wiki/Geki)*
+
+*Diharapkan tidak keliru dengan Katu dari osu!taiko*
```

Comment:
```suggestion
*Berbeda konteks dengan Katu dari osu!taiko*
```
As referenced here https://osu.ppy.sh/wiki/id/Disambiguation, i think it should be changed to `*Berbeda konteks dengan Katu dari osu!taiko*`.